packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/usePaymentInformation.js

```diff
@@ -105,6 +113,19 @@ export const usePaymentInformation = props => {
         [onSave]
     );
 
+    const clearPaymentDetails = useCallback(() => {
```

Comment:
This will make sure even if the user refreshes the page, the summary won't be shown because `hasData` will be false.